tests/golden/apiserver/appcat/appcat/10_provider_sql.yaml

```diff
@@ -45,18 +45,6 @@ metadata:
     name: crossplane-provider-provider-sql-system-custom
   name: crossplane:provider:provider-sql:system:custom
 rules:
-  - apiGroups:
```

Comment:
This rule was invalid to begin with. So it's not actually needed, because it never worked in the first place.